app/controllers/comments_controller.rb

```diff
@@ -1,10 +1,10 @@
 class CommentsController < ApplicationController
   before_action :set_comment, only: %i[ show edit update destroy ]
   before_action :is_an_authorized_user, only: [:destroy, :create]
+  before_action { authorize @comment || Comment }
```

Comment:
Your before_action saved a lot of code, thanks to it we no longer need to check authorize @comment every time! 👍

Reply:
+1